test.js

```diff
@@ -1,5 +1,5 @@
 import test from 'ava';
-import libui from '.';
+import libui from './index';
```

Comment:
Why this change? require('.') should be independent by the cwd and equivalent to require('./index.js')